.github/actions/setup-default-test-properties/test-properties.json

```diff
@@ -14,7 +14,7 @@
   },
   "JavaTestProperties": {
     "SUPPORTED_VERSIONS": ["8", "11", "17"],
-    "FLINK_VERSIONS": ["1.13", "1.14", "1.15"],
```

Comment:
just a side note

this is stale implies this field is not used, actually currently this action is only used in update_python_dependencies and publish_github_release_notes workflow.

Currently there is hard coded python/java versions everywhere in our GHAs. As futurre clean up we could make use of this action to have single source of version setting for actions